### PR TITLE
Added normalized scalar for canvas drawing, added Box drawing flags, moved target string inside Render class

### PR DESCRIPTION
**Pull request by 64blit**
Additionally, a "dev" script in the Renderer2d that adds a build system on .ts file changes, as well as a node/vite server that refreshes on any live changes, allowing for quick testing.

` cd src/eyepop-render-2d; npm run dev `
`cd example/vite; npm run dev` > http://localhost:5173/

-------------
Squashed commits:

[update] Moved target string inside Renderer interface and added dynamic canvas scalar

[update] added unreasonably large file testing

[fix] added back rect width based scaling, to help with small bounds

[fix] added sensible hdpi scaling

[fix] normalized width of lines

[update] Added normalized font scaling

[update] Added dev script to build on any file changes

[fix] fixed text offset for multiple nested labels

[update] Added drawing flags for finer control

[update] Moved target string inside Renderer interface and added dynamic canvas scalar to handle large media

[update] Added scale factor to account for large images

[update] moved target string into Render interface

[create] added larger test file

[update] Moved target string inside Renderer interface and added dynamic canvas scalar

### CODE REVIEW

File: examples/vite/.gitignore
Comment (tschulz) [inline examples/vite/.gitignore:1]: Why do we need another .gitignore in a subfolder?

File: examples/vite/public/example.jpg [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/edmunddao/_SPACE/EyePop.ai/Untitled/eyepop-sdk-node/examples/example.jpg
```
Comment (tschulz) [inline examples/vite/public/example.jpg:1]: This seems to be an erroneous commit.

File: examples/vite/public/large_example.jpg [hunk 2]
```diff
@@ -0,0 +1 @@
+/Users/edmunddao/_SPACE/EyePop.ai/Untitled/eyepop-sdk-node/examples/large_example.jpg
```
Comment (tschulz) [inline examples/vite/public/large_example.jpg:1]: ditto

File: examples/vite/src/eyepop.min.js [hunk 3]
```diff
@@ -0,0 +1 @@
+/Users/edmunddao/_SPACE/EyePop.ai/Untitled/eyepop-sdk-node/src/eyepop/dist/eyepop.min.js
```
Comment (tschulz) [inline examples/vite/src/eyepop.min.js:1]: what are those files?

File: examples/vite/src/vite-env.d.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (tschulz) [inline examples/vite/src/vite-env.d.ts:1]: is that an empty file?

**Review by tschulz — CHANGES_REQUESTED** [review]

File: examples/vite/src/vite-env.d.ts (hunk 4 above)
Comment (64blit) [reply examples/vite/src/vite-env.d.ts]: You are correct - has been deleted

File: examples/vite/src/eyepop.min.js (hunk 3 above)
Comment (64blit) [reply examples/vite/src/eyepop.min.js]: They were symlinks used for testing - I removed them and added them directly to the server

`    resolve: {
        alias: {
            'eyepop': path.resolve(__dirname, '../../src/eyepop/dist/eyepop.min.js'),
            'eyepop-render-2d': path.resolve(__dirname, '../../src/eyepop-render-2d/dist/eyepop.render2d.min.js'),
            'example.jpg': path.resolve(__dirname, '../../examples/example.jpg'),
            'large_example.jpg': path.resolve(__dirname, '../../examples/large_example.jpg'),
        }
    },`

https://github.com/eyepop-ai/eyepop-sdk-node/pull/31/commits/1b14f6251cf82dc803abbcdb14a2710bb675b917

File: examples/vite/public/example.jpg (hunk 1 above)
Comment (64blit) [reply examples/vite/public/example.jpg]: Also a sym link, resolved via [1b14f62](https://github.com/eyepop-ai/eyepop-sdk-node/pull/31/commits/1b14f6251cf82dc803abbcdb14a2710bb675b917)

File: examples/vite/.gitignore
Comment (64blit) [reply examples/vite/.gitignore:1]: Indeed! File has been deleted https://github.com/eyepop-ai/eyepop-sdk-node/pull/31/commits/512452a08ed3e663cf54e446e8d789c4c2cd4d8c

**Comment by 64blit** [comment]
Added relative box padding and removed confidence label:

image

**Review by tschulz — APPROVED** [review]

**Pull request merged**